src/components/flat-table/flat-table-row/flat-table-row.spec.js

```diff
@@ -106,29 +106,43 @@ describe("FlatTableRow", () => {
               <FlatTableRowHeader>test 1</FlatTableRowHeader>
               <FlatTableHeader>test 2</FlatTableHeader>
               <FlatTableCell>test 3</FlatTableCell>
+              <FlatTableRowHeader stickyAlignment="right">
+                test 4
+              </FlatTableRowHeader>
             </FlatTableRow>
           </thead>
         </table>
       );
 
       assertStyleMatch(
         {
-          borderLeft: "2px solid var(--colorsSemanticFocus500)",
-          zIndex: "1002",
+          borderLeft: "3px solid var(--colorsSemanticFocus500)",
         },
         wrapper,
-        { modifier: `:focus ${StyledFlatTableRowHeader}` }
+        {
+          modifier: `:focus ${StyledFlatTableRowHeader}:first-of-type:not(:last-of-type)::before`,
+        }
+      );
+
+      assertStyleMatch(
+        {
+          borderRight: "2px solid var(--colorsSemanticFocus500)",
```

Comment:
I noticed in this file we are verifying the golden focus border as 2px wide instead of 3px which is used elsewhere in Carbon. I appreciate one pixel difference is incredibly minor - so if this change is not complex, could you update the styling for `StyledFlatTableRowHeader` so its consistent with Design System?